src/routes/knowledge/[id]/+page.svelte

```diff
@@ -54,11 +55,18 @@
 			</Button>
 		</p>
 		<p data-testid="knowledge-timestamp" class="text-sm text-muted">
-			{#if isNewKnowledge}
-				New knowledge
+			<!-- {#if isNewKnowledge}
+			New knowledge
 			{:else}
-				{knowledge.updatedAt}
-			{/if}
+			<Badge variant="default" capitalize={false}>
+				<p data-testid="knowledge-timestamp">
+					{formatShortDistanceToNow(new Date(knowledge.updatedAt), { addSuffix: true })}
+					</p>
+					</Badge>
+					{/if} -->
+			{isNewKnowledge
+				? 'New knowledge'
+				: formatDistanceToNow(new Date(knowledge.updatedAt), { addSuffix: true })}
```

Comment:
We now have several instances of `formatDistanceToNow(...)` so let's add a new function in `src/lib/utils.ts` called something like `export function formatTimestamp(date: string)` and replace all other instances with this new function.